pkgs/development/python-modules/pyscf/default.nix

```diff
@@ -0,0 +1,70 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cmake
+, openblas
+, libcint
+, libxc
+, h5py
+, numpy
+, scipy
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "pyscf";
+  version = "1.7.0";
+
+  # must download from GitHub to get the Cmake & C source files
+  src = fetchFromGitHub {
+    owner = "pyscf";
+    repo = "pyscf";
+    rev = "v${version}";
+    sha256 = "0gdmc04nr699bq6gkw9isfzahj0k2gqhxnjg6gj9rybmglarkn15";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [
+    libcint
+    libxc
+    openblas
+  ];
+  cmakeFlags = [
+    # disable rebuilding/downloading the required libraries
+    "-DBUILD_LIBCINT=0"
+    "-DBUILD_LIBXC=0"
+    "-DBUILD_XCFUN=0"
+  ];
+  dontUseCmakeConfigure = true;
+
+  propagatedBuildInputs = [
+    h5py
+    numpy
+    scipy
+  ];
+
+  PYSCF_INC_DIR = "${libcint}/lib:${libxc}/lib";
+
+  doCheck = true;
+  pythonImportsCheck = [ "pyscf" ];
+  checkInputs = [ pytest ];
+  checkPhase = ''
+    pytest ./pyscf/tools/test
+  '';
+
+  meta = {
+    description = ''
+      PySCF: Python-based Simulations of Chemistry Framework
+    '';
```

Comment:
one line please. Also dont need to say the package name, it's already in context of the package

```
description = "Python-based Simulations of Chemistry Framework";
```